Activating extension streetsidesoftware.code-spell-checker failed due to an error: Cannot find module 'settings-viewer-next'

Cannot load the extension 

Extension verision is 2.17.0
VS Code version is 1.75.1

```text
2023-02-20 18:44:02.696 [error] Activating extension streetsidesoftware.code-spell-checker failed due to an error:
2023-02-20 18:44:02.696 [error] Error: Cannot find module 'settings-viewer-next'
Require stack:
- c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js
- c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js
- c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-amd.js
- c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-fork.js
- 
	at Module._resolveFilename (node:internal/modules/cjs/loader:987:15)
	at i._resolveFilename (node:electron/js2c/utility_init:17:840)
	at Module._load (node:internal/modules/cjs/loader:832:27)
	at c._load (node:electron/js2c/asar_bundle:5:13343)
	at r._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:122:13360)
	at g._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:117:62400)
	at E._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:117:61793)
	at Module.require (node:internal/modules/cjs/loader:1059:19)
	at p (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:4:699)
	at 8098 (c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:741430)
	at n (c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:854157)
	at 6574 (c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:282757)
	at n (c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:854157)
	at c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:854769
	at Object.<anonymous> (c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:854795)
	at u._compile (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:4:1313)
	at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
	at Module.load (node:internal/modules/cjs/loader:1035:32)
	at Module._load (node:internal/modules/cjs/loader:876:12)
	at c._load (node:electron/js2c/asar_bundle:5:13343)
	at r._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:122:13360)
	at g._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:117:62400)
	at E._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:117:61793)
	at Module.require (node:internal/modules/cjs/loader:1059:19)
	at require (node:internal/modules/cjs/helpers:102:18)
	at Function.r [as __$__nodeRequire] (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:5:101)
	at f.pb (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:117:63664)
	at async Promise.all (index 0)
```

--- a/packages/client/src/extension.ts
+++ b/packages/client/src/extension.ts
@@ -15,7 +15,7 @@ import { ConfigTargetLegacy, sectionCSpell } from './settings';
 import { initStatusBar } from './statusbar';
 import { logErrors, silenceErrors } from './util/errors';
 import { performance } from './util/perf';
-import { activate as settingsViewerActivate } from 'settings-viewer-next';
+// import { activate as settingsViewerActivate } from 'settings-viewer-next';
 
 performance.mark('cspell_done_import');
 
@@ -24,7 +24,7 @@ modules.init();
 export async function activate(context: ExtensionContext): Promise<ExtensionApi> {
     performance.mark('cspell_activate_start');
 
-    await settingsViewerActivate(context);
+    // await settingsViewerActivate(context);
 
     // Get the cSpell Client
     const client = await CSpellClient.create(context);

